### PR TITLE
Add pywinrm

### PR DESCRIPTION
**Pull request by synapticarbors**
Depends on https://github.com/conda-forge/staged-recipes/pull/4173

### CODE REVIEW

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes I think this is ready to merge. I'd appreciate if someone could take a quick look.

File: recipes/pywinrm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - requests_ntlm >=0.3.0
+
+test:
+  imports:
+    - winrm
+
+about:
+  home: https://github.com/diyan/pywinrm
+  license: MIT
+  license_family: MIT
```
Comment (isuruf) [inline recipes/pywinrm/meta.yaml]: No license file included?

Comment (synapticarbors) [reply recipes/pywinrm/meta.yaml]: Added issue upstream:

https://github.com/diyan/pywinrm/issues/183

Comment (synapticarbors) [reply recipes/pywinrm/meta.yaml]: manually packaged the license file in latest commit.

**Pull request merged**